### PR TITLE
[fr] SUREMENT_ACCENT

### PR DESCRIPTION
**Pull request by remymartinDev**

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -4240,6 +4240,16 @@ though, and has since been slightly modified)-->
...
+            <pattern>
+                <marker>
+                    <token>surement</token>
+                </marker>
+            </pattern>
+                <message>Ce mot s'écrit avec un accent circonflexe.</message>
+            <suggestion>sûrement</suggestion>
+            <example correction="sûrement">Tu as <marker>surement</marker> raison.</example>
+        </rule>
         <rule id="POLE_2" name="pôle" default="temp_off">
```
Comment (jaumeortola) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:4253]: "Surement" is "orthographe rectifiée de 1990". See: https://fr.wiktionary.org/wiki/surement
Perhaps we need coherency rules or user settings (new/traditional orthography).

Comment (remymartinDev) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:4253]: Should I add "En orthographe traditionnelle (non réformée), attention à l'accent circonflexe." in the message ?
Or should I drop this rule ?

Comment (jaumeortola) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:4253]: As far as I know, we just accept all forms (traditional and reformed). I don't know if there are plans for other things. @udomai

**Comment by remymartinDev** [comment]
Rule abandonned -> irrelevant

**Pull request closed without merging**